lib/internal/fs/streams.js

```diff
@@ -392,22 +393,73 @@ WriteStream.prototype.open = openWriteFs;
 
 WriteStream.prototype._construct = _construct;
 
+function writeAll(data, size, pos, cb, retries = 0) {
+  this[kFs].write(this.fd, data, 0, size, pos, (er, bytesWritten, buffer) => {
+    if (er?.code === 'EAGAIN') {
```

Comment:
Can you add a comment about what this error code represents?

Reply:
@anonrig it is documented here: https://github.com/nodejs/node/blob/60d7cf3541eefe726c20dc13e5562fe94fd7eeef/doc/api/os.md#L705-L708
Anyway. would `// No data currently available and operation should be retried later.` be a good explanation?

Reply:
Yes, perfect